README.md

```diff
@@ -211,6 +211,18 @@ To remove Dapr from your Kubernetes cluster, use the `uninstall` command with `-
 $ dapr uninstall -k
 ```
 
+### Upgrade Dapr on Kubernetes
+
+To perform a zero downtime upgrade of the Dapr control plane:
```

Comment:
Make a note of the release name being `dapr` and if installed using helm, should use helm to upgrade.

Reply:
Added a note for the installation, where it belongs.